docs/wallet_browser_extension/definition.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+  - wallet
+  - browser extension
+  - provider API
+---
+
+# Definition
```

Comment:
This should be `Defintions`, to match with the content page

Reply:
solved by 86f2f28